src/streaming.ts

```diff
@@ -175,16 +174,20 @@ export class StreamProcessor {
       const msg = JSON.parse(line.substring(5));
 
       if (msg.domain === 'flag') {
+        debugStreamEventReceived(this.log);
```

Comment:
Curious why you moved this inside the `if`'s? It _might_ be useful to log when messages are not a known `domain`. Not a blocker at all.

Reply:
Cool - it was to get rid of that pesky heartbeat log. But, I've updated it so we only log messages if they have a `domain` property, e.g. flags/segments/others but not heartbeats. https://github.com/harness/ff-nodejs-server-sdk/commit/f8a2c99878ba56207f8ec2c447014981552fb803